### ISSUE TITLE
pip doesn't work well if the version is specified in `ja-ginza-dict`.

### ISSUE THREAD

**Issue by pesuchin** [megagonlabs/ginza#109]
## TL;DR

When I try to install ginza using Poetry which is a package management tool, the installation fails with an error.

The cause of this is that pip doesn't work well if the version is specified in `ja-ginza-dict`.

I'm sorry, but I could not find the root cause.
If you can afford it, I would appreciate it if you could respond this issue.

## environment

- MacOS Catalina 10.15.4
- Poetry version 1.0.2
- Python 3.7.6

## steps to reproduce

I tried to install ginza with the following command, but I could not install it with the following error.

```
$ poetry init  # All enter
$ poetry add ginza
Using version ^3.1.2 for ginza

Updating dependencies
Resolving dependencies... (1.6s)

Writing lock file


Package operations: 7 installs, 0 updates, 0 removals

  - Installing cython (0.29.17)
  - Installing dartsclone (0.9.0)
  - Installing ja-ginza-dict (3.1.0)

[EnvCommandError]
Command ['/Path/to/venv/.venv/bin/pip', 'install', '--no-deps', 'ja-ginza-dict==3.1.0'] errored with the following return code 1, and output: 
ERROR: Could not find a version that satisfies the requirement ja-ginza-dict==3.1.0 (from versions: 3.1.0.post1)
ERROR: No matching distribution found for ja-ginza-dict==3.1.0
```

## additional information

Also, the same error occurred with the following command.

```
$ pip install ja-ginza-dict==3.1.0
ERROR: Could not find a version that satisfies the requirement ja-ginza-dict==3.1.0 (from versions: 3.1.0.post1)
ERROR: No matching distribution found for ja-ginza-dict==3.1.0
```

However, the following command succeeds in the installation.

```
$ pip install ja-ginza-dict
or
$ pip install ja-ginza-dict==3.1.0.post1
```

**Comment by hiroshi-matsuda-rit** [comment]
Thank you @pesuchin 
I test two pattern of the version specification.

> pip install ja-ginza-dict==3.1.0 -> failed
> pip install ja-ginza-dict>=3.1.0 -> succeded

There might be a problem on PyPI or in our package file.
I'd like to research this issue by next release.

By the way, SudachiPy team just released latest dictionary packages from PyPI, today.
We may change dictionary package of GiNZA by using this official package.

**Comment by hiroshi-matsuda-rit** [comment]
From GiNZA version 4.0.0, we changed installing archive of SudachiDict-core from proprietary to the official one (from PyPI).
Could you test the GiNZA v4? @pesuchin
